ci/infra/libvirt/terraform.tfvars.json.ci.example

```diff
@@ -1,9 +1,9 @@
 {
-    "libvirt_uri": "qemu:///system",
+    "libvirt_uri": "qemu+ssh://jenkins@node52.cloud.suse.de/system",
     "pool": "default",
-    "image_uri": "SLES15-SP1-JeOS.x86_64-15.1-OpenStack-Cloud-QU1.qcow2",
+    "image_uri": "http://download.suse.de/ibs/SUSE:/SLE-15-SP1:/GA/images/SLES15-SP1-JeOS.x86_64-15.1-OpenStack-Cloud-Build35.30.qcow2",
```

Comment:
We need to agree on which image to use and the URI should be static, this one is depending on the build.